### PR TITLE
Html Project

### PR DESCRIPTION
**Pull request by Abbas4tech**
Created Simple Web page using html and css basics

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/02-Project/Style.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+h4 {
+    font-size: 15px;
+}
+p {
+    font-size: 20px;
+    color: brown;
+}
+p, h3 {
+    padding: 0 0 0 20px;
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/02-Project/Style.css:28]:
```suggestion
    padding-left: 20px;
```

File: tasks-by-users/AbbasShaikh0304/02-Project/Style.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    color: yellow;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.button {
+    display: block;
+    text-decoration: none;
+    width: 150px;
+    height: 50%;
+    padding: 10px;
+    margin: 0 0 0 20px;
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/02-Project/Style.css:51]:
```suggestion
    margin-left: 20px;
```

File: tasks-by-users/AbbasShaikh0304/02-Project/Style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+* {
+    background-color: burlywood;
+}
+img {
+    width: 100%;
+    height: 500px;
+    justify-content: center;
```
[the remark this answers is not in the document]
Comment (faran4engg) [reply tasks-by-users/AbbasShaikh0304/02-Project/Style.css:7]: nice catch @mujeeb100

File: tasks-by-users/AbbasShaikh0304/02-Project/index.html [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+        The
+        Categories of the Followers of <a href="https://en.wikipedia.org/wiki/Al-Shafi%27i">Imam
+            Shafi</a>.
+    </p>
+    <hr>
+    <h4>you may translate the Tafsir of each Surah by using Bing Translator in many lanuages; Just right click
+        on your
+        mouse and Click to "translate with Bing"</h4>
+    <hr>
+    <ul class="Lists">
```
Comment (faran4engg) [inline tasks-by-users/AbbasShaikh0304/02-Project/index.html:48]: class names should be in lower case....

**Pull request merged**